File: C#/LojaBrinquedos/LojaBrinquedos/Pagamento.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace LojaBrinquedos
{
    public class Pagamento
    {
        public Pagamento() { }

        public Pagamento(int identificacao, string tipoPagamento) {
            this.Identificacao = identificacao;
            this.tipoPagamento = tipoPagamento;
        }
       public int Identificacao { get; set; }
       public string tipoPagamento { get; set; }
    }
}

File: Java/ClassesObjetos/src/Pagamento.java

public class Pagamento {
	private int identificacao;
	private String tipoPagamento;
	
	public Pagamento() {}
	
	public Pagamento(int identificacao, String tipoPagamento) {
		this.identificacao = identificacao;
		this.tipoPagamento = tipoPagamento;
	}
	
	public void setIdentificacao(int identificacao) {
		this.identificacao = identificacao;
	}
	
	public int getIdentificacao() {
		return this.identificacao;
	}
	
	public void setTipoPagamento(String tipoPagamento) {
		this.tipoPagamento = tipoPagamento;
	}
	
	public String getTipoPagamento() {
		return this.tipoPagamento;
	}
}
